SourceX/qol.cpp

```diff
@@ -0,0 +1,377 @@
+/**
+ * @file qol.cpp
+ *
+ * Quality of life features
+ */
+#include "all.h"
+#include "../3rdParty/Storm/Source/storm.h"
+
+DEVILUTION_BEGIN_NAMESPACE
+
+int drawMinX;
+int drawMaxX;
+int highlightItemsMode = 0;
+// 0 = disabled
+// 1 = highlight when alt pressed
+// 2 = hide when alt pressed
+// 3 = always highlight
+bool altPressed = false;
+bool drawXPBar = false;
+bool drawHPBar = false;
+
+int GetTextWidth(char *s)

```

Comment:
`int GetTextWidth(const char *s)` could avoid some "warning: ISO C++ forbids converting a string constant to 'char*'"